Client/Components/ComparisonComponent.razor.cs

```diff
@@ -221,5 +232,14 @@ public float SetBarWidth()
             float maxValue = ValueMap.Values.Max(valueList => (float)valueList.Max(dataPoint => dataPoint.Value));
             return maxValue * 1.3f;
         }
+
+        public async Task OpenBiggerViewOfGraph()
+        {
+            ComparisonComponent copy = this;
```

Comment:
This is not a copy of the component. It is just the same reference.  Would it be better to pass the values or something and create a new graph?